strapi/strapi
# WIP[admin]: Exp-418 Audit logs list api mapping

### What does it do?

- Removed mock data and mapped with api 'admin/audit-logs'

### How to test it?

- Dashboard > Settings > Admin Logs
- User should be able to see logs with changes done inside dashboard, example, try creating new role, add new entry etc.

### Related issue(s)/PR(s)

- Waiting for this BE PR to merge before: [PR](https://github.com/strapi/strapi/pull/15286#discussion_r1058828520)
